### ISSUE TITLE
KCP Network has Error and Ready condition

### ISSUE THREAD

**Issue by tmilos77** [kyma-project/cloud-manager#853]
**Description**

KCP Networks with both Error and Ready condition exist. For example following is observed in cases when Scope exists
```yaml
  conditions:
    - message: Ready
      reason: Ready
      status: 'True'
      type: Ready
    - message: Scope xxxxxxxxx does not exist
      reason: ScopeNoFound
      status: 'True'
      type: Error
  state: Ready
```
Additionally when Network is changed with an annotation both conditions remain

**Expected result**

KCP Network should have either Ready or Error state and not both of them.

**Actual result**

**Steps to reproduce**

**Troubleshooting**

**Comment by tmilos77** [comment]
PR #862

**Comment by tmilos77** [comment]
Solved with [#852](https://github.com/kyma-project/cloud-manager/issues/852)
